src/test/typescript/typescript-preparser_test.ts

```diff
@@ -53,10 +53,10 @@ suite('TypeScriptParser', () => {
 
     test('throws a WarningCarryingException for parse errors', async () => {
       const contents = 'const const const const const #!@(~~)!();';
-      const url = 'ts-parse-error.ts';
+      const url = 'ts-parse-error.ts' as ResolvedUrl;
```

Comment:
nit: `rUrl`? Or is that something we want to implicitly test here

Reply:
+1, changed